kura/org.eclipse.kura.core.net/src/main/java/org/eclipse/kura/core/net/util/NetworkUtil.java

```diff
@@ -72,11 +50,7 @@ public static short getNetmaskShortForm(String netmask) throws KuraException {
             throw new KuraException(KuraErrorCode.INTERNAL_ERROR, "netmask is null");
```

Comment:
@cdealti Cool. No Problem. But I believe a null parameter check is a runtime problem which the client is not meant to handle to take precautions. And, that's why a null parameter check must incorporate an unchecked exception translation.

If you consider any null parameter check or any specific length check of an array or collection param, clients ain't meant to write precautionary codes to deal with such conditions. For a checked exception translation, we have to make sure that the exceptional conditions must necessitate clients to take precautions in case these conditions occur.